logging/Cargo.toml

```diff
@@ -8,5 +8,8 @@ rust-version.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-env_logger.workspace = true
 log.workspace = true
+thiserror.workspace = true
+tokio = { workspace = true, default-features = false, features = ["rt", "sync"] }
```

Comment:
Tokio with logging... what could go wrong...

Just for the record, I hate this... but I think I can't provide a case against it to stop it. I hope I'm wrong.